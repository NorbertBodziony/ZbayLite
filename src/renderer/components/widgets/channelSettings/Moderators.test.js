import React from 'react'
import { shallow } from 'enzyme'
import Immutable from 'immutable'

import { Moderators } from './Moderators'
import { mockClasses } from '../../../../shared/testing/mocks'

describe('Moderators', () => {
  it('renders component', () => {
    const result = shallow(
      <Moderators
        classes={mockClasses}
        users={Immutable.Map({})}
        moderators={Immutable.List()}
        openAddModerator={() => {}}
        removeModerator={() => {}}
      />
    )
    expect(result).toMatchSnapshot()
  })
})
