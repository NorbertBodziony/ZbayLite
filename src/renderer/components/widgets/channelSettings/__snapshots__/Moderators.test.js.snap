// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Moderators renders component 1`] = `
<AutoSizer
  disableHeight={false}
  disableWidth={false}
  onResize={[Function]}
  style={Object {}}
>
  <Component />
</AutoSizer>
`;
